src/main/java/se/sundsvall/smloader/integration/db/model/CaseMapping.java

```diff
@@ -30,16 +30,13 @@ public class CaseMapping {
 	private String externalCaseId;
 
 	@Id
-	@Column(name = "caseId")
-	private String caseId;
+	@Column(name = "errandId")
+	private String errandId;
 
 	@NotNull
 	@Column(name = "caseType", columnDefinition = "varchar(255)")
 	private String caseType;
 
-	@Column(name = "serviceName")
-	private String serviceName;
-
 	@Schema(accessMode = Schema.AccessMode.READ_ONLY)
 	@Column(name = "timestamp")
```

Comment:
Why is this attribute called "timestamp"?

"timestamp" is a data type, the name should not describe the data type, but rather express the purpose of the attribute. For example. "created", "modified", "sent" and so on...